Makefile

```diff
@@ -94,8 +106,18 @@ test-unit-coverage: test-unit
 	gocov report build/coverage/coverprofile.json
 
 .PHONY: test-e2e
-test-e2e: crds
-	GO111MODULE=on TEST_WATCH_NAMESPACE=${TEST_NAMESPACE} ginkgo ${TEST_E2E_FLAGS} test/e2e
+test-e2e: crds test-e2e-plain
+
+.PHONY: test-e2e-plain
+test-e2e-plain:
+	GO111MODULE=on \
+	TEST_OPERATOR_NAMESPACE=${TEST_NAMESPACE} \
```

Comment:
Since we have `.EXPORT_ALL_VARIABLES` set, you don't need to set the same variables again.

Reply:
Hi @otaviof, I did this based on a review comment I got earlier from @qu1queee in another PR. In his opinion it improves readability and I can understand this. Note that for `TEST_NAMESPACE` specifically, the assignment is needed because we still use the old `TEST_NAMESPACE` variable and assign this to the new `TEST_WATCH_NAMESPACE` and `TEST_OPERATOR_NAMESPACE` variables as long as we have not yet the need to provide separate values.

Reply:
Well, in this case, fair enough. Thanks.